cmd/nvidia-dra-plugin/device_state.go

```diff
@@ -349,35 +357,70 @@ func (s *DeviceState) prepareDevices(ctx context.Context, claim *resourceapi.Res
 
 func (s *DeviceState) unprepareDevices(ctx context.Context, claimUID string, devices PreparedDevices) error {
 	for _, group := range devices {
-		// Stop any MPS control daemons started for each group of prepared devices.
-		mpsControlDaemon := s.mpsManager.NewMpsControlDaemon(claimUID, group)
```

Comment:
Why does it seem as if this was removed?

Reply:
hmm it shouldn't be. I'll put it back